project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,492 @@
+        apply_logic:
+          - and:
+            - not: Windows|No Windows
+            - not: Windows|Triple, Low-E, Insulated, Argon, H-Gain
+            - not: Windows|Triple, Low-E, Insulated, Argon, L-Gain
+            - or:
+              - ASHRAE IECC Climate Zone 2004|4A
+              - ASHRAE IECC Climate Zone 2004|4B
+              - ASHRAE IECC Climate Zone 2004|4C
+        costs:
+          - value: 45.77 # TBD
+            multiplier: Window Area (ft^2)
+        lifetime: 30 #TBD
+      - &EnergyStar_window_IECC_CZ5A-5B-6A-6B-7A-7B
+        option: Windows|EnergyStar, Northern
+        apply_logic:
+          - and:
+            - not: Windows|No Windows
+            - or:
+              - ASHRAE IECC Climate Zone 2004|5A
+              - ASHRAE IECC Climate Zone 2004|5B
+              - ASHRAE IECC Climate Zone 2004|6A
+              - ASHRAE IECC Climate Zone 2004|6B
+              - ASHRAE IECC Climate Zone 2004|7A
+              - ASHRAE IECC Climate Zone 2004|7B
+        costs:
+          - value: 45.77 # TBD
+            multiplier: Window Area (ft^2)
+        lifetime: 30 #TBD  
+
+  - upgrade_name: Light Touch Envelope
+    options:
+      #Attic floor insulation up to IECC 2021 levels for homes with vented attics and less than R-13 existing insulation
+      - &attic_insulation_IECC_CZ1A
+        option: Insulation Ceiling|R-30
+        apply_logic:
+          - and:
+            - ASHRAE IECC Climate Zone 2004|1A
+            - Geometry Attic Type|Vented Attic
+            - or:
+              - Insulation Ceiling|Uninsulated
+              - Insulation Ceiling|R-7
+              - Insulation Ceiling|R-13
+        costs:
+          - value: 0.0443 # $/(ft^2 * Delta R-value), from original REMDB xlsx for blown cellulose (0.0154+0.019), with 1.289 inflation factor
+            multiplier: Floor Area, Attic * Insulation Increase (ft^2 * Delta R-value)
+          - value: 0.1992 # $/(ft^2), from original REMDB xlsx for blown cellulose (0.1505+0.004), with 1.289 inflation factor
+            multiplier: Floor Area, Attic (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf 
+      - &attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+        option: Insulation Ceiling|R-49
+        apply_logic:
+          - and:
+            - or:
+              - ASHRAE IECC Climate Zone 2004|2A
+              - ASHRAE IECC Climate Zone 2004|2B
+              - ASHRAE IECC Climate Zone 2004|3A
+              - ASHRAE IECC Climate Zone 2004|3B
+              - ASHRAE IECC Climate Zone 2004|3C
+            - Geometry Attic Type|Vented Attic
+            - or:
+              - Insulation Ceiling|Uninsulated
+              - Insulation Ceiling|R-7
+              - Insulation Ceiling|R-13
+              - Insulation Ceiling|R-19
+              - Insulation Ceiling|R-30
```

Comment:
missing `- Insulation Ceiling|R-38`

Reply:
Yes, it is intentional.
The intent document is to apply to dwelling units with R-30 or less insulation. I refer to the previous EUSS run. I'm not sure whether it is correct.

Reply:
(As I had said elsewhere, just adding it here, let's add R-38)